whitesands/code/modules/vending/mining_equipment.dm

```diff
@@ -1,131 +0,0 @@
-/* Mining Points Equipment Vendor */
-/* Mining Points Equipment Vendor */
-/obj/machinery/vending/mining_equipment
-	name = "frontier equipment vendor"
-	desc = "An equipment vendor for miners, prospectors, and all manner of far reach scroungers. Ore Redemption Points can be spent here to purchase rough-and-tumble goods. Sold by EXOCON."
-	icon = 'whitesands/icons/obj/machines/vending.dmi'
```

Comment:
Hey since we're removing the vendors, mind deleting the sprites too? (I sincerely doubt we will ever need these again)
It's not used anywhere else.